dbt_server/server.py

```diff
@@ -83,4 +84,5 @@ async def startup_event():
     # This method is `async` intentionally to block serving until startup is complete
     configure_uvicorn_access_log()
     dbt_service.inject_dd_trace_into_core_lib()
-    startup_cache_initialize()
+    if DBT_CLOUD_CONTEXT.get() != "develop":
+        startup_cache_initialize()
```

Comment:
Only needed for SL?

Reply:
Correct, this is doing a dbt-server /parse and creating those state folders, which we don't use for the IDE. I definitely look forward to removing that whole StateController once we make an IDE-specific version of this code!